src/auth-service/utils/email.msgs.js

```diff
@@ -143,7 +143,7 @@ module.exports = {
   user_updated: (firstName, lastName) => {
     return (
       `Dear ${firstName + " " + lastName} \n\n` +
```

Comment:
correcting a typo in te emails for when user details are updated. @BenjaminSsempala